### PR TITLE
Fix error 500

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: config/database.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'DB_DATABASE' => __DIR__ . '/../database/database.sqlite',
```
Comment (samsonasik) [inline config/database.php:4]: @TomasVotruba this is needed due to now require absoute path.

File: .env.dist [hunk 2]
```diff
@@ -5,4 +5,3 @@ APP_KEY=base64:c9yprqAy56EwJ1pvPeyRkP6xV3MTtsa498Hsffd3/rw=
 APP_URL=http://localhost
 
 DB_CONNECTION=sqlite
-DB_DATABASE=database/database.sqlite
```
Comment (samsonasik) [inline .env.dist:8]: the relative path need to updated to use absolute path on config/database.php.

**Comment by samsonasik** [comment]
Ref https://github.com/rectorphp/rector/issues/8422

still error, seems some permission issue, I am still looking into it.

**Comment by samsonasik** [comment]
It fixed now 👍

File: config/database.php (hunk 1 above)
Comment (samsonasik) [reply config/database.php:4]: It seems only define database type on .env is enough, and will use database/database.sqlite, see:

- https://github.com/rectorphp/getrector-com/pull/1963

**Comment by TomasVotruba** [comment]
Hi Abdul, thanks for quick fix. Do you know what it was caused by?
I think I didn't lock Laravel to specific commit, and there was a BC break.

**Comment by samsonasik** [comment]
It was a php-fpm down, which result 503, restarting it got no path cache, updating laravel got 500, it was rough guess

**Pull request merged**